app/konnect/declarative-config.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Managing a Kong-hosted control plane with declarative config
+no_version: true
+---
+
+Manage entities in your Konnect cloud control plane with declarative
+configuration. To do this, you need [decK](/deck/).
+
+With declarative configuration, you can manage:
+* **All parts of a service:** Service versions, implementations, routes, and
+plugins.
+* **Dev Portal documents:** Specs and markdown files.
+* **Global configuration:** If you want to use global plugins or consumers, you
+have to use declarative configuration. Global plugins appear in the
+Konnect UI as read-only.
+
+DecK _cannot_ manage Dev Portal, application registration, or custom plugins.
+
+## Prerequisites
+
+* Permissions?
+* decK is [installed](/deck/installation/).
+
+## Test your connection
+
+First, make sure that you can log in to Konnect and that decK recognizes your
+account credentials:
+
+```sh
+deck konnect ping --konnect-email <email> --konnect-password <password>
+```
+
+If the connection is successful, the terminal displays the first and last name
+attributed to the account:
+
+```sh
+Successfully Konnected as Some Name (Kong)!
+```
+
+{:.tip}
+> **Tip:** You can also use decK with Konnect more securely by storing your
+password in a file, then calling it with `--konnect-password-file <pass>.txt`.
```

Comment:
The wording might need work, but I think referencing the decK config option as well might be helpful:

```suggestion
> **Tip:** You can also use decK with Konnect more securely by storing your
password in a file, then calling it with either `--konnect-password-file 
<pass>.txt` or by adding it to your decK configuration under the
`konnect-password` option.
```